eng/Directory.Build.Common.props

```diff
@@ -1,4 +1,22 @@
 ﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- Setup project type properties -->
+  <PropertyGroup>
+    <IsMgmtLibrary Condition="'$(IsMgmtClientLibrary)' == '' and ($(MSBuildProjectName.StartsWith('Azure.Management.')) or $(MSBuildProjectName.StartsWith('Azure.ResourceManager.')))">true</IsMgmtLibrary>
```

Comment:
by this point $(MSBuildProjectName.StartsWith('Azure.Management.')) should never be true right?

Reply:
Nope I don't see any projects named that way so I'll remove that.